xamarin/Xamarin.Forms
# [iOS] Fix issue rendering gradient background in NavigationPage

### Description of Change ###

Fix issue rendering gradient background in NavigationPage on iOS.

### Issues Resolved ### 

- fixes #14380 

### API Changes ###
 
 None

### Platforms Affected ### 

- iOS

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 

Not applicable

### Testing Procedure ###
Launch Core Gallery and navigate to the issue 14380. If can see the gradient background, the test has passed.

### PR Checklist ###
